client/my-sites/theme/main.jsx

```diff
@@ -9,6 +9,7 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import page from 'page';
+import { isPremium } from 'my-sites/themes/helpers';
```

Comment:
Yeah this will be fixed by #5259.

`import { isPremium, getForumUrl } from 'my-sites/themes/helpers';`

Incidentally, the above doesn't work since `getForumUrl` uses `this.isPremium()`.